libvast/src/system/spawn_archive.cpp

```diff
@@ -39,10 +40,13 @@ maybe_actor spawn_archive(node_actor* self, spawn_arguments& args) {
   auto mss
     = 1_MiB
       * get_or(args.inv.options, "max-segment-size", sd::max_segment_size);
-  auto actor = self->spawn(archive, args.dir / args.label, segments, mss);
+  auto handle = self->spawn(archive, args.dir / args.label, segments, mss);
+  if (!handle)
```

Comment:
I don't understand why we need those checks here. I thought local spawn can't fail at runtime?

Reply:
Alright, removed them again.